lib/kion/car.go

```diff
@@ -108,19 +108,19 @@ func GetCARSOnAccount(host string, token string, accID uint) ([]CAR, error) {
 	return cars, nil
 }
 
-// GetCARByNameAndAccount returns a car that matches a given name for a given account number
-func GetCARByName(host string, token string, carName string) (CAR, error) {
+// GetCARByNameAndAccount returns a CAR that matches a given name and account number
+func GetCARByNameAndAccount(host string, token string, carName string, accountNumber string) (CAR, error) {
```

Comment:
So this was my first approach but the Kion API is not returning a full list of car/account combinations in some situations. For example in #7 the user has minimal project permissions. In that scenario we made a test users that had one car assigned to three accounts. The API only returned one CAR with one account causing this setup to fail.

Can you show me how to replicate the issue this is attempting to solve?

Reply:
See issue `9996` internally.

Reply:
Ok I think the solution is to revert the changes to this func, and inject the account number directly into the returned car within  the favorites func in `main.go`. See the suggestion there.